api/alert_channel_datadog_test.go

```diff
@@ -0,0 +1,152 @@
+//
+// Author:: Vatasha White (<vatasha.white@lacework.net>)
+// Copyright:: Copyright 2021, Lacework Inc.
+// License:: Apache License, Version 2.0
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+package api_test
+
+import (
+	"fmt"
+	"net/http"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+
+	"github.com/lacework/go-sdk/api"
+	"github.com/lacework/go-sdk/internal/intgguid"
+	"github.com/lacework/go-sdk/internal/lacework"
+)
+
+func TestAlertChannelsService_GetDatadog(t *testing.T) {
+	var (
+		intgGUID    = intgguid.New()
+		apiPath     = fmt.Sprintf("AlertChannels/%s", intgGUID)
+		fakeServer  = lacework.MockServer()
+		APIKey      = "Im-a-fake-api-key"
+		DatadogSite = "com"
+		DatadogType = "Logs Detail"
+	)
+	fakeServer.UseApiV2()
+	fakeServer.MockToken("TOKEN")
+	defer fakeServer.Close()
+
+	fakeServer.MockAPI(apiPath, func(w http.ResponseWriter, r *http.Request) {
+		assert.Equal(t, "GET", r.Method, "GetDatadog should be a GET method")
+		fmt.Fprintf(w, generateAlertChannelResponse(singleDatadogAlertChannel(intgGUID)))
+	})
+
+	c, err := api.NewClient("test",
+		api.WithApiV2(),
+		api.WithToken("TOKEN"),
+		api.WithURL(fakeServer.URL()),
+	)
+	assert.Nil(t, err)
+
+	response, err := c.V2.AlertChannels.GetDatadog(intgGUID)
+	assert.Nil(t, err)
+	assert.NotNil(t, response)
```

Comment:
Couldn't avoid noticing that your https://github.com/lacework/go-sdk/pull/565#discussion_r721497598 there applies to your codebase over here. We should update both. cc @dmurray-lacework thoughts?

Reply:
Changed it @afiune!